frontend/src/ui/blocks/Series.tsx

```diff
@@ -32,6 +34,25 @@ type GridTypeProps = {
 
 const GridTile: React.FC<GridTypeProps> = ({ event, realmPath }) => {
     const [THUMB_WIDTH, THUMB_HEIGHT] = [16, 9].map(x => x * 15);
+    const thumbnail = event.thumbnail == null
+        ? <div css={{
+            display: "flex",
+            width: THUMB_WIDTH,
+            height: THUMB_HEIGHT,
+            borderRadius: 4,
+            backgroundColor: "var(--grey92)",
+            alignItems: "center",
+            justifyContent: "center",
+            fontSize: 40,
+        }}>
+            <FontAwesomeIcon icon={faVolumeUp} />
```

Comment:
I don't know that we can assume a missing thumbnail to imply an audio-only event. In fact I know that we generally can't, because you can write arbitrary workflows for Opencast.